docs/source/Support/bskReleaseNotes.rst

```diff
@@ -104,6 +104,8 @@ Version 2.2.1 (Dec. 22, 2023)
 - Marked the method ``CreateNewTask()`` input variable ``InputDelay`` as depreciated.  This variable
   was never implemented and did nothing.
 - Fixed terminal events to terminate at the time they are triggered instead of one timestep after.
+- Added a coast option to the :ref:`prescribedRot1DOF` module, where a period of zero acceleration is added between
+  the two acceleration segments.
```

Comment:
This comment needs to be move to the current BSK version release notes :-)

Reply:
Done